### ISSUE TITLE
live preview from partials

### ISSUE THREAD

**Issue by Turqueso** [jasonsanjose/brackets-sass#91]
I'm not sure if this is normal o if i'm explaining myself right, so bear with me please.

I have some similar structure
```
index.html
stylesheets
  -main.scss
  -main.css
  -components
           --_variables.scss
           --_header.scss
```

If I have a main file with all my partials imported, when modifying a partials, changes won't compile until save; so live preview won't show until save.

**Comment by ofhouse** [comment]
+1
Would be very useful when a partial is changed that the main.scss would be automatically re-compiled.

**Comment by jasonsanjose** [comment]
@Turqueso I think this may have been a windows bug where the source map paths were set incorrectly, causing the link from the main file to the partial file to be lost. 

You can try the latest dev build here https://github.com/jasonsanjose/brackets-sass/releases/tag/2.0.1-101 (download the zip, open extension manager, then drag the zip from your file system to the drop zone).

If you have problems uninstalling, try the workaround here https://github.com/jasonsanjose/brackets-sass/issues/81#issuecomment-76223733.

**Comment by Turqueso** [comment]
@jasonsanjose the latest dev build fixed my issue with compiling partials. thanks!
But there is not livepreview when i make changes on partials, i don't know if this is supposed to work on this build

**Comment by jasonsanjose** [comment]
Fixed in #100. Closing.
